demos/lib/domvm-mobx.js

```diff
@@ -117,24 +121,28 @@ function becameStale(vm) {
 
 // The diff.eq() assigned to each observer vm:
 function eq(vm) {
-	var observerData = vm.mobxObserver;
+	var observerData = vm._mobxObserver;
 	
-	if (observerData.stale) return false;	// Re-render.
-	else if(observerData.eq) return observerData.eq.apply(this, arguments); // Let diff() choose.
-	else return true;	// By default: no re-render.
+	if (observerData.stale)
+		return false;	// Re-render.
+	else if(observerData.eq)
```

Comment:
@jlgrall i added a follw-up commit to add the space between the `if(`

Reply:
Thx for telling me.